package.json

```diff
@@ -85,15 +91,21 @@
     "@types/ws": "^4.0.1",
     "@types/yargs": "^8.0.2",
     "angular": "^1.7.4",
+    "archiver": "^3.0.0",
     "babel-core": "^6.26.0",
+    "babel-plugin-syntax-object-rest-spread": "^6.13.0",
+    "brfs": "^1.6.1",
+    "browserify": "^16.2.3",
     "bundlesize": "^0.14.4",
     "chalk": "^2.4.1",
     "codecov": "^2.2.0",
     "commitizen": "^2.9.6",
     "conventional-changelog-cli": "^1.3.4",
     "coveralls": "^2.11.9",
+    "cpy": "^7.0.1",
```

Comment:
not a big deal, but I generally preferred to use https://github.com/shelljs/shelljs instead of bringing in all the individual commands (cpy,del,make-dir,etc) with different APIs to remember

Reply:
yeah, I think we should consider that for the future. `del` and `make-dir` bring in a crazy number of deps :) Some of the simpler bash scripts could switch over as well so they work cross platform.